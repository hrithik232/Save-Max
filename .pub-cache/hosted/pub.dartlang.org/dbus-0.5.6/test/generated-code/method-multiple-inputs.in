<node>
  <interface name="com.example.Test">
    <method name="Hello">
      <arg name="byte_value" type="y" direction="in"/>
      <arg name="boolean_value" type="b" direction="in"/>
      <arg name="int16_value" type="n" direction="in"/>
      <arg name="uint16_value" type="q" direction="in"/>
      <arg name="int32_value" type="i" direction="in"/>
      <arg name="uint32_value" type="u" direction="in"/>
      <arg name="int64_value" type="x" direction="in"/>
      <arg name="uint64_value" type="t" direction="in"/>
      <arg name="double_value" type="d" direction="in"/>
      <arg name="string_value" type="s" direction="in"/>
      <arg name="object_path_value" type="o" direction="in"/>
      <arg name="signature_value" type="g" direction="in"/>
      <arg name="variant_value" type="v" direction="in"/>
      <arg name="struct_value" type="(si)" direction="in"/>
      <arg name="array_value" type="ay" direction="in"/>
      <arg name="dict_value" type="a{sv}" direction="in"/>
    </method>
  </interface>
</node>
