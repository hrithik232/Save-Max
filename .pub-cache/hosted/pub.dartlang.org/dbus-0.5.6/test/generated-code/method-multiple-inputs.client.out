import 'package:dbus/dbus.dart';

class ComExampleTest extends DBusRemoteObject {
  ComExampleTest(DBusClient client, String destination, DBusObjectPath path) : super(client, destination, path);

  /// Invokes com.example.Test.Hello()
  Future<void> callHello(int byte_value, bool boolean_value, int int16_value, int uint16_value, int int32_value, int uint32_value, int int64_value, int uint64_value, double double_value, String string_value, String object_path_value, DBusValue signature_value, DBusValue variant_value, DBusStruct struct_value, List<int> array_value, Map<String, DBusValue> dict_value, {bool noAutoStart = false, bool allowInteractiveAuthorization = false}) async {
    await callMethod('com.example.Test', 'Hello', [DBusByte(byte_value), DBusBoolean(boolean_value), DBusInt16(int16_value), DBusUint16(uint16_value), DBusInt32(int32_value), DBusUint32(uint32_value), DBusInt64(int64_value), DBusUint64(uint64_value), DBusDouble(double_value), DBusString(string_value), DBusObjectPath(object_path_value), signature_value, DBusVariant(variant_value), struct_value, DBusArray.byte(array_value), DBusDict.stringVariant(dict_value)], replySignature: DBusSignature(''), noAutoStart: noAutoStart, allowInteractiveAuthorization: allowInteractiveAuthorization);
  }
}
